typescript/infra/src/config/gas-oracle.ts

```diff
@@ -112,6 +116,15 @@ function getLocalStorageGasOracleConfigOverride(
   }, {});
 }
 
+export function getTypicalRemoteGasAmount(
+  local: ChainName,
+  remote: ChainName,
+  getOverhead: (local: ChainName, remote: ChainName) => number,
+): number {
+  const handleGasUsage = 50_000;
```

Comment:
worth making this a public const somewhere if we reuse 50k somewhere else?
```suggestion
  const handleGasUsage = 50_000;
```